examples/cross-contract-high-level/src/lib.rs

```diff
@@ -41,14 +41,15 @@ pub trait ExtStatusMessage {
 
 #[near_bindgen]
 impl CrossContract {
-    pub fn deploy_status_message(&self, account_id: AccountId, amount: U128) {
-        Promise::new(account_id)
+    pub fn deploy_status_message(&self, account_id: AccountId, amount: U128) -> Promise {
+        let new_account_id = format!("{}.{}", account_id, env::current_account_id())
```

Comment:
this line should throw a compiler error since there's no semicolon at the end

Reply:
```suggestion
        let new_account_id = format!("{}.{}", account_id, env::current_account_id());
```